xml/obs_best_practice_staging_workflow.xml

```diff
@@ -49,34 +49,70 @@
     </figure>
     <itemizedlist>
       <listitem>
-        <para>
-          <emphasis>Managers:</emphasis>
-          Shows the group that are the Staging Mangers
-        </para>
-      </listitem>
-      <listitem>
-        <para>
-          <emphasis>Empty projects:</emphasis>
-          Staging projects without and requests assigned.
-        </para>
-      </listitem>
-      <listitem>
-        <para>
-          <emphasis>Backlog:</emphasis>
-          List of requests that can be assigned to a staging project.
-        </para>
-      </listitem>
-      <listitem>
-        <para>
-          <emphasis>Ready:</emphasis>
-          List of requests that were in the backlog and have a review accepted.
-        </para>
+      <para>
+        Table content:
+      </para>
+        <itemizedlist>
+          <listitem>
+            <para>
+              <emphasis>Staging Project:</emphasis>
+              Shows the staging project name, it's overall state (see legend),
+              and the overall build progress of the packages within the project.
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              <emphasis>Requests:</emphasis>
+              Shows the different requests associated and the current state of them.
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              <emphasis>Problems:</emphasis>
+              Shows build problems of packages within the project and status problems
+              from external services like openQA sended to the status api of the buildservice.
```

Comment:
[...] and status problems reported to the Build Service's Status API by external services like openQA.